src/bindings/jna/libelektra4j/src/main/java/org/libelektra/PluginLoader.java

```diff
@@ -0,0 +1,40 @@
+package org.libelektra;
+
+import org.libelektra.exception.InstallationException;
+import org.libelektra.plugin.Echo;
+import org.libelektra.plugin.NativePlugin;
+import org.libelektra.plugin.PropertiesStorage;
+import org.libelektra.plugin.Return;
+
+/**
+ * This class can be used to load Plugins from Elektra.
+ * It also loads self implemented Java plugins.
+ */
+public class PluginLoader {
+
+	private Key errorKey;
+
+	public PluginLoader(Key errorKey) {
```

Comment:
Javadoc still missing. Why is this constructor needed? You can also instantiate the errorKey internally?

Reply:
Developers should also have the possibility to define the errorKey themselve in my opinion to access its properties.

Should I add a no-arg constructor which with the default error key name `system/jna/default/errorkey`?

Reply:
> Should I add a no-arg constructor which with the default error key name system/jna/default/errorkey?

Yes, but I would leave the name empty.